### DISCUSSION TITLE
Retrieving a similar list of packages via API as the one shown on the GUI

### DISCUSSION THREAD

**Discussion by ricardogsilva** [ckan/ckan#6481] [Q&A]
The `/api/3/action/package_list` API endpoint does not show the same content as the dataset list shown when visiting the CKAN `/datasets` page. Is this intentional?

In order to get the same results via API I seem to need to use the `/api/3/package_search` endpoint, and send it the `include_private=True` parameter.

As a user, it seems odd that the `package_list` API endpoint would not show me private datasets that I have access to, while the `package_search` endpoint does. Not sure if this should become an issue, or maybe it is just a CKAN design decision?

**Comment by frafra** [comment]
That looks like a bug to me. If you are authenticated, then there is no reason why `package_list` should not show you the private datasets.
